modules/meb_api/lib/dgi/eligibility/configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'dgi/configuration'
+
+module MebApi
+  module DGI
+    module Eligibility
+      class Configuration < MebApi::DGI::Configuration
+        def base_path
+          Settings.dgi['veteran-services'].url.to_s
```

Comment:
Same comments here as the modules/meb_api/lib/dgi/automation/configuration.rb file.